README.md

```diff
@@ -1,167 +1,281 @@
+     --no-password --insecure \
+     root.linkerd.cluster.local \
+     keys/ca.crt keys/ca.key
 ```
 
-12. Add the `linkerd-control-plane` helm chart as an application:
+Generate the identity issuer certificate and key:
+
+```bash
+#@immed
+rm -f keys/issuer.crt keys/issuer.key
+step certificate create \
+     --profile intermediate-ca --not-after 8760h \
+     --ca keys/ca.crt --ca-key keys/ca.key \
+     --no-password --insecure \
+     identity.linkerd.cluster.local \
+     keys/issuer.crt keys/issuer.key
 ```
-argocd app create linkerd-control-plane     --repo https://helm.linkerd.io/stable     --helm-chart linkerd-control-plane     --revision 1.16.2     --dest-namespace linkerd     --dest-server https://kubernetes.default.svc     --helm-set identityTrustAnchorsPEM="$(cat ca.crt)"     --helm-set identity.issuer.tls.crtPEM="$(cat issuer.crt)"     --helm-set identity.issuer.tls.keyPEM="$(cat issuer.key)"
+
+<!-- @wait_clear -->
+
+### Create the `linkerd` namespace
+
+We'll also create the `linkerd` namespace by hand, rather than having Argo CD
+do it:
+
+```bash
+kubectl create namespace linkerd
 ```
 
-13. Sync both the applications.
+### Add the `linkerd-crd` app
+
+We'll tell Argo CD to use Helm to install Linkerd for us, so we'll start by
+using `argocd` to register the Linkerd Helm chart repo:
+
+```bash
+argocd repo add https://helm.linkerd.io/stable --type helm --name linkerd-stable
 ```
-`argocd app sync linkerd-crd linkerd-control-plane`
+
+Once that's done, we can define an Argo CD application for the `linkerd-crds`
+Helm chart, version 1.8.0. This corresponds to Linkerd `stable-2.14.0`.
+
+```bash
+argocd app create linkerd-crds \
+       --repo https://helm.linkerd.io/stable \
+       --helm-chart linkerd-crds \
+       --revision 1.8.0 \
+       --dest-namespace linkerd \
+       --dest-server https://kubernetes.default.svc
 ```
-You should see them both in sync and healthy.
 
-14. Get a list of your CRDs
+<!-- @wait_clear -->
+
+### Add the `linkerd-control-plane` app
+
+This is the same as before, but for the Linkerd control plane itself. We use
+the `linkerd-control-plane` chart, version 1.16.2, which is (again) Linkerd
+`stable-2.14.0`. We also explicitly provide the certificates here, as values
+for the chart.
+
+```bash
+argocd app create linkerd-control-plane \
+       --repo https://helm.linkerd.io/stable \
+       --helm-chart linkerd-control-plane \
+       --revision 1.16.2 \
+       --dest-namespace linkerd \
+       --dest-server https://kubernetes.default.svc \
+       --helm-set identityTrustAnchorsPEM="$(cat keys/ca.crt)" \
+       --helm-set identity.issuer.tls.crtPEM="$(cat keys/issuer.crt)" \
+       --helm-set identity.issuer.tls.keyPEM="$(cat keys/issuer.key)"
 ```
-kubectl get crds
+
+<!-- @wait_clear -->
+
+### Sync both applications
+
+This tells Argo CD to go ahead and make the cluster look like the applications
+we've defined.
+
+```bash
+argocd app sync linkerd-crd linkerd-control-plane
```

Comment:
this should be `argocd app sync linkerd-crds linkerd-control-plane`  (with an S at the end of linkerd-crds), I'll merge this and make a quick PR to fix it